massa-execution/src/types.rs

```diff
@@ -249,6 +249,16 @@ impl EventStore {
     }
 }
 
+#[derive(Clone)]
+pub struct StackElement {
+    /// called address
+    pub address: Address,
+    /// coins transferred to the target address during a call,
+    pub coins: Amount,
+    /// list of addresses created so far during excution,
+    pub owned_addresses: Set<Address>,
```

Comment:
this was set to be a vec (and not a set) for a good reason: we need to keep deterministic ordering

Reply:
I did that because of the specification in the issue

Reply:
there was a followup overriding this: https://github.com/massalabs/massa/pull/2181